packages/cashier-v2/src/lib/Transfer/components/TransferForm/components/TransferAccountSelection/TransferAccountSelection.tsx

```diff
@@ -35,7 +40,17 @@ const TransferAccountSelection = () => {
     return (
         <div className={styles.container}>
             <TransferDropdown accounts={accounts} label='From' onSelect={onSelectFromAccount} value={fromAccount} />
-            <TransferDropdown accounts={filteredToAccounts} label='To' onSelect={onSelectToAccount} value={toAccount} />
+            <TransferDropdown
+                accounts={filteredToAccounts}
+                label='To'
+                message={
+                    accountLimits?.daily_transfers && fromAccount?.account_type
+                        ? `You have ${fromAccountLimit} transfers remaining for today.`
+                        : ''
+                }
```

Comment:
```suggestion
                message={
                    accountLimits?.daily_transfers && fromAccount?.account_type
                        ? `You have ${fromAccountLimit} transfers remaining for today.`
                        : ''
                }
```
could we extract it to separate message variable?